sdks/python/container/piputil.go

```diff
@@ -0,0 +1,132 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Contains the utility functions to install python pip packages.
+package main
+
+import (
+  "bufio"
+  "bytes"
+  "errors"
+  "fmt"
+  "io/ioutil"
+  "log"
+  "path/filepath"
+  "strings"
+
+  "github.com/apache/beam/sdks/go/pkg/beam/util/execx"
+)
+
+const (
+  pip = "/usr/local/bin/pip"
+)
+
+// PipInstallRequirements installs the given requirement, if present.
+func PipInstallRequirements(files []string, dir, name string) error {
```

Comment:
nit: these functions do not need to be exported now (= start with uppercase). They can just be private.